core/src/operations/decorators/advice.rs

```diff
@@ -30,6 +30,11 @@ pub enum AdviceInjector {
     /// Given an element of quadratic extension field, it computes multiplicative inverse and
     /// injects the result into advice tape.
     Ext2Inv,
+
+    /// Given a polynomial ( in NTT form ) of domain size power of 2, this routine interpolates
+    /// ( using inverse NTT ) the polynomial to get a polynomial in coefficient form, which
+    /// injected into advice tape.
```

Comment:
We should update this in the same way as in the decorators file.

Reply:
Sorry missed that, fixed in 3f9de5858fb847aa6e866a3eac121f05e4ddef68